src/VirtualClient/VirtualClient.Actions/SPECview/SpecViewExecutor.cs

```diff
@@ -71,6 +81,28 @@ public string Viewset
             }
         }
 
+        /// <summary>
+        /// Using PsExec to run specviewperf in session 1.
+        /// </summary>
+        public string PsExecPackageName
+        {
+            get
+            {
+                return this.Parameters.GetValue<string>(nameof(SpecViewExecutor.PsExecPackageName));
+            }
+        }
+
+        /// <summary>
+        /// PsExec session number
+        /// </summary>
+        public int PsExecSession
```

Comment:
Can we do this:
Default to -1 or empty, and run without psexec, if this is passed in, then we use psexec with that number. We need to do the same thing with 3dmark as well

Reply:
I will open a separate PR for 3dmark. It needs a few other changes too.